package.json

```diff
@@ -2106,6 +2106,13 @@
 						"scope": "window",
 						"order": 11
 					},
+					"gitlens.graph.enableGhostRefs": {
```

Comment:
I think we should align this name with `highlightRowsOnRefHover` -- so maybe `gitlens.graph.showGhostRefsOnRowHover`

Reply:
Done in commit: `Modified name of "enableGhostRefs" property to be "showGhostRefsOnRowHover" instead`